### PR TITLE
Added Local Store In Syncman Package

### PR DESCRIPTION
**Pull request by prithvi081099**

### CODE REVIEW

**Comment by sharadregoti** [comment]
Task linked: [CU-2vtg8w Add Syncman Store Local](https://app.clickup.com/t/2vtg8w)

**Comment by sharadregoti** [comment]
@YourTechBud Review this pr. I have tested this with pr with previous version config file & it works.

File: gateway/utils/syncman/syncman.go [hunk 1]
```diff
@@ -84,49 +87,49 @@ func New(nodeID, clusterID, advertiseAddr, storeType, runnerAddr, configFile str
...
-			if s.projectConfig.Projects != nil && len(s.projectConfig.Projects) > 0 {
-				s.modules.SetProjectConfig(s.projectConfig, s.letsencrypt, s.routing)
-			}
-		}); err != nil {
-			return err
+	// Start routine to admin config
+	if err := s.store.WatchAdminConfig(func(clusters []*config.Admin) {
+		s.lock.Lock()
+		defer s.lock.Unlock()
+		logrus.WithFields(logrus.Fields{"admin config": clusters}).Debugln("Updating admin config")
```
Comment (YourTechBud) [inline gateway/utils/syncman/syncman.go:115]: Use utils.logdebug

**Comment by sharadregoti** [comment]
>     * test if the realtime is still working on single node instance
> 
>     * make sure the store config do not conflict sith enterprise version @sharadregoti

do I have to test the realtime feature on a Kubernetes cluster or docker ?

**Comment by YourTechBud** [comment]
On docker. This code is changing behaviour of single node clusters

File: gateway/utils/server/server.go [hunk 2]
```diff
@@ -66,13 +66,15 @@ func New(nodeID, clusterID, advertiseAddr, storeType, runnerAddr, configFile str
...
-
+func (s *Server) Start(isDev, profiler bool, staticPath string, port int, restrictedHosts []string, ssl *config.SSL) error {
 	// Start the sync manager
-	if err := s.syncMan.Start(s.syncMan.GetGlobalConfig(), port); err != nil {
+	if err := s.syncMan.Start(ssl, port); err != nil {
 		return err
 	}
 
+	// Configure all modules
+	s.SetConfig(s.syncMan.GetGlobalConfig(), !isDev)
```
[the remark this answers is not in the document]
Comment (sharadregoti) [reply gateway/utils/server/server.go:76]: we need this, without this, 
we won't be able to change dev mode of sc.
the admin man will not get cluster config stored in config.yaml

**Pull request merged**